### ISSUE TITLE
Accented vowels are not rendered correctly

### ISSUE THREAD

**Issue by kalbun** [Microsoft/vscode#1838]
Letters like "è" or "ù" are rendered with a question mark - see attached image.
Using Windows 10 with Italian layout and keyboard.

![cattura](https://cloud.githubusercontent.com/assets/7243235/12155356/d85bf822-b4c4-11e5-8b21-13a9631775a3.PNG)

**Comment by alexandrudima** [comment]
@kalbun Does this happen when you type "è" or "ù" in a new file or when you open a specific file. If this happens when you open a specific file, it might be that the file is using a different encoding. The status bar shows the current encoding the file is opened with and allows you to open it with a different encoding:

![image](https://cloud.githubusercontent.com/assets/5047891/12321142/e50cde5e-baac-11e5-9422-2128b0fd0fec.png)

**Comment by fabiogusmao** [comment]
This happened to me last week while editing a legacy ASP application. I changed the company's logo url on template.asp and users started complaining about **accented words showing incorrectly** (ie: "Usuários became Usu?rios). Back then I realized VSCode changed the file's encoding from **ISO-8859-1** to **UTF-8**.

I believe this is a bug and VSCode should check the file's encoding while opening it and keep that encoding.
